src/components/LearningMaterial/index.tsx

```diff
@@ -27,7 +27,7 @@ export type Props = {
   publishedAt?: string
   updatedAt?: string
   locale: Locale
-  courseCopy?: CoursePageCopy
+  landingPageCopy?: CoursePageCopy
```

Comment:
This should not be optional. But I get that you keep it that way because lecturepage and blockpage don't have their copy yet. We need to make sure to remember to make this mandatory once we have all three in place.